includes/class-woopay-tracker.php

```diff
@@ -541,7 +541,7 @@ public function checkout_order_processed( $order_id ) {
 		$properties      = [ 'payment_title' => 'other' ];
 
 		// If the order was placed using WooCommerce Payments, record the payment title using Tracks.
-		if ( strpos( $payment_gateway->id, 'woocommerce_payments' ) === 0 ) {
+		if ( $payment_gateway && strpos( $payment_gateway->id, 'woocommerce_payments' ) === 0 ) {
```

Comment:
what about checking if the variable is defined?

```suggestion
		if ( isset( $payment_gateway ) && strpos( $payment_gateway->id, 'woocommerce_payments' ) === 0 ) {
```